[BUG] BuildConfig is defined multiple times

### Important Instructions ⚠️

Before submitting this issue, please **read carefully** and confirm the following:

- [x] I am using the **latest version** of `sherpa-onnx`. If not, I will upgrade and check if the issue persists.
- [x] I understand that **issues not reproducible on the latest version** may be closed without response.
- [x] I have verified that the error is reproducible using **only `sherpa-onnx`**, not code from my own project.
- [ ] I will provide **detailed steps** to reproduce the issue.
- [x] I will provide **complete logs** and error messages.
- [x] I understand that **issues not following these instructions** may be closed or receive no response.

---

### Describe the issue

`flutter build apk --release` Build fails on Android with version sherpa_onnx 1.12.40 . Downgrade to 1.12.34 build success

### Steps to reproduce

1. I can not create reproduce source, in my project i upgrade sherpa_onnx from 1.12.34  to 1.12.40 and it build fails on Android

### Expected behavior

Build flutter Android success with sherpa_onnx 1.12.40

### Environment
- `sherpa-onnx` version: 1.12.40
- OS: macOS 26.4.1
- Python version: 3.11.9
- onnxruntime version:
- GPU (if applicable):
- NPU (if applicable):

### Logs
```bash
ERROR: /Volumes/SamsungDisk/sources/build/sherpa_onnx_android_arm64/intermediates/runtime_library_classes_jar/release/bundleLibRuntimeToJarRelease/classes.jar: R8: Type com.k2fsa.sherpa.onnx.BuildConfig is defined multiple times: /Volumes/SamsungDisk/build/sherpa_onnx_android_arm64/intermediates/runtime_library_classes_jar/release/bundleLibRuntimeToJarRelease/classes.jar:com/k2fsa/sherpa/onnx/BuildConfig.class, /Volumes/SamsungDisk/build/sherpa_onnx_android_armeabi/intermediates/runtime_library_classes_jar/release/bundleLibRuntimeToJarRelease/classes.jar:com/k2fsa/sherpa/onnx/BuildConfig.class

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:minifyReleaseWithR8'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.R8Task$R8Runnable
   > Compilation failed to complete, origin: /Volumes/SamsungDisk/build/sherpa_onnx_android_arm64/intermediates/runtime_library_classes_jar/release/bundleLibRuntimeToJarRelease/classes.jar:com/k2fsa/sherpa/onnx/BuildConfig.class
```

--- a/flutter/notes.md
+++ b/flutter/notes.md
@@ -30,10 +30,10 @@ flutter create --template plugin_ffi --platforms linux sherpa_onnx_windows
 5. Create `sherpa_onnx_android_arm64`, `sherpa_onnx_android_armeabi`, `sherpa_onnx_android_x86`, `sherpa_onnx_android_x86_64`
 
 ```bash
-flutter create --template plugin_ffi --platforms android --org com.k2fsa.sherpa.onnx sherpa_onnx_android_arm64
-flutter create --template plugin_ffi --platforms android --org com.k2fsa.sherpa.onnx sherpa_onnx_android_armeabi
-flutter create --template plugin_ffi --platforms android --org com.k2fsa.sherpa.onnx sherpa_onnx_android_x86
-flutter create --template plugin_ffi --platforms android --org com.k2fsa.sherpa.onnx sherpa_onnx_android_x86_64
+flutter create --template plugin_ffi --platforms android --org com.k2fsa.sherpa.onnx.arm64 sherpa_onnx_android_arm64
+flutter create --template plugin_ffi --platforms android --org com.k2fsa.sherpa.onnx.armeabi sherpa_onnx_android_armeabi
+flutter create --template plugin_ffi --platforms android --org com.k2fsa.sherpa.onnx.x86 sherpa_onnx_android_x86
+flutter create --template plugin_ffi --platforms android --org com.k2fsa.sherpa.onnx.x86_64 sherpa_onnx_android_x86_64
 ```
 
 6. Create `sherpa_onnx_ios`

--- a/flutter/sherpa_onnx_android_arm64/android/build.gradle
+++ b/flutter/sherpa_onnx_android_arm64/android/build.gradle
@@ -25,7 +25,7 @@ rootProject.allprojects {
 apply plugin: "com.android.library"
 
 android {
-    namespace 'com.k2fsa.sherpa.onnx'
+    namespace 'com.k2fsa.sherpa.onnx.arm64'
 
     // Bumping the plugin compileSdk version requires all clients of this plugin
     // to bump the version in their app.

--- a/flutter/sherpa_onnx_android_arm64/android/src/main/AndroidManifest.xml
+++ b/flutter/sherpa_onnx_android_arm64/android/src/main/AndroidManifest.xml
@@ -1,3 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.k2fsa.sherpa.onnx">
+  package="com.k2fsa.sherpa.onnx.arm64">
 </manifest>

--- a/flutter/sherpa_onnx_android_armeabi/android/build.gradle
+++ b/flutter/sherpa_onnx_android_armeabi/android/build.gradle
@@ -25,7 +25,7 @@ rootProject.allprojects {
 apply plugin: "com.android.library"
 
 android {
-    namespace 'com.k2fsa.sherpa.onnx'
+    namespace 'com.k2fsa.sherpa.onnx.armeabi'
 
     // Bumping the plugin compileSdk version requires all clients of this plugin
     // to bump the version in their app.

--- a/flutter/sherpa_onnx_android_armeabi/android/src/main/AndroidManifest.xml
+++ b/flutter/sherpa_onnx_android_armeabi/android/src/main/AndroidManifest.xml
@@ -1,3 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.k2fsa.sherpa.onnx">
+  package="com.k2fsa.sherpa.onnx.armeabi">
 </manifest>

--- a/flutter/sherpa_onnx_android_x86/android/build.gradle
+++ b/flutter/sherpa_onnx_android_x86/android/build.gradle
@@ -25,7 +25,7 @@ rootProject.allprojects {
 apply plugin: "com.android.library"
 
 android {
-    namespace 'com.k2fsa.sherpa.onnx'
+    namespace 'com.k2fsa.sherpa.onnx.x86'
 
     // Bumping the plugin compileSdk version requires all clients of this plugin
     // to bump the version in their app.

--- a/flutter/sherpa_onnx_android_x86/android/src/main/AndroidManifest.xml
+++ b/flutter/sherpa_onnx_android_x86/android/src/main/AndroidManifest.xml
@@ -1,3 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.k2fsa.sherpa.onnx">
+  package="com.k2fsa.sherpa.onnx.x86">
 </manifest>

--- a/flutter/sherpa_onnx_android_x86_64/android/build.gradle
+++ b/flutter/sherpa_onnx_android_x86_64/android/build.gradle
@@ -25,7 +25,7 @@ rootProject.allprojects {
 apply plugin: "com.android.library"
 
 android {
-    namespace 'com.k2fsa.sherpa.onnx'
+    namespace 'com.k2fsa.sherpa.onnx.x86_64'
 
     // Bumping the plugin compileSdk version requires all clients of this plugin
     // to bump the version in their app.

--- a/flutter/sherpa_onnx_android_x86_64/android/src/main/AndroidManifest.xml
+++ b/flutter/sherpa_onnx_android_x86_64/android/src/main/AndroidManifest.xml
@@ -1,3 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="com.k2fsa.sherpa.onnx">
+  package="com.k2fsa.sherpa.onnx.x86_64">
 </manifest>

